src/Compilers/Core/Portable/InternalUtilities/ArrayExtensions.cs

```diff
@@ -26,6 +26,12 @@ internal static T[] Copy<T>(this T[] array, int start, int length)
             return newArray;
         }
 
+        public static int IndexOf<T>(this T[] array, T value)
+            => Array.IndexOf(array, value);
+
+        public static bool Contains<T>(this T[] array, T value)
+            => Array.IndexOf(array, value) >= 0;
```

Comment:
These helpers avoid the following errors during bootstrap:

```
src\Compilers\Core\Portable\CommandLine\CommandLineParser.cs(1068,45): error CS0121: (NETCORE_ENGINEERING_TELEMETRY=Build) The call is ambiguous between the following methods or properties: 'System.MemoryExtensions.IndexOf<T>(System.Span<T>, T)' and 'Roslyn.Utilities.EnumerableExtensions.IndexOf<T>(System.Collections.Generic.IEnumerable<T>, T)'
src\Compilers\Core\Portable\MetadataReader\MetadataHelpers.cs(305,73): error CS0121: (NETCORE_ENGINEERING_TELEMETRY=Build) The call is ambiguous between the following methods or properties: 'System.MemoryExtensions.Contains<T>(System.Span<T>, T)' and 'System.Linq.Enumerable.Contains<TSource>(System.Collections.Generic.IEnumerable<TSource>, TSource)'
```

See also test `BreakingChange_ExtensionMethodLookup_SpanVsIEnumerable`. We might need to add a betterness rule to avoid this break.